README.md

```diff
@@ -58,10 +64,42 @@ Don't start this too quickly after starting the flight controller and Unreal. MA
 
 To use PX4, add the arg `ardupilot:=false`. 
 
-The camera ROS publishers are disabled by default as it slows the publishing rate for everything to ~3Hz. To run with camera publishers on, add the arg `enable_cameras:=true`. When running with cameras enabled, Rviz looks like this.
+The camera ROS publishers are disabled by default as it slows the publishing rate for everything to ~3Hz. To run with camera publishers on, add the arg `enable_cameras:=true`. When running with cameras enabled and do_octomap set to true, Rviz looks like this.
 
 ![](images/airsim-start-ros.png)
 
+### manual flight
+
+To test how the map looks as the vehicle moves around, you can fly the ArduPilot drone with terminal commands. In the ArduPilot terminal, run:
+
+```
+mode guided
+arm throttle
+takeoff 2
+module load message
+```
+Then you can use any of the following commands.
+
+* message SET_POSITION_TARGET_LOCAL_NED 0 0 0 9 3576 20 0 0 0 0 0 0 0 0 0 0
+    * Flies 20m forward (necessary in Blocks to see an obstacle)
+* message SET_POSITION_TARGET_LOCAL_NED 0 0 0 9 3576 0 0 -2 0 0 0 0 0 0 0 0
+    * Flies up 5m
+* message SET_POSITION_TARGET_LOCAL_NED 0 0 0 9 2503 0 0 0 0 0 0 0 0 0 1.5707963 0
+    * Rotates 90 deg to right
+* message SET_POSITION_TARGET_OFFSET_NED 0 0 0 9 3527 0 0 0 2 0 0 0 0 0 0 0
```

Comment:
Is this supposed to be OFFSET_NED not LOCAL?

Reply:
yes, they are offsets relative to the vehicle, so you can keep sending the same command and it'll go another 20m forward every time

Reply:
I just mean why is this particular message on L89 'OFFSET' instead of 'LOCAL'?

Reply:
oh! yes that was a mistake, good catch! just pushed the corrected command